testsuite/forge.py

```diff
@@ -689,11 +629,29 @@ def get_dashboard_link(
         if "forge" in forge_cluster_name
         else INTERN_GRAFANA_BASE_URL
     )
-    return f"{base_url}&var-namespace={forge_namespace}&var-chain_name={forge_chain_name}{grafana_time_filter}"
+    return (
+        f"{base_url}&var-namespace={forge_namespace}"
```

Comment:
Can we use some kind of URL + query builder for this, this looks like a footgun angle.

Reply:
Yes, if you think this is a footgun you should look at humio link

Reply:
I think this is worth following up on but not blocking this PR

Reply:
Open an issue pls pls 😼